minoc/README.md

```diff
@@ -72,22 +87,17 @@ org.opencadc.minoc.authenticateOnly=true
 With `authenticateOnly=true`, any authenticated user will be able to read/write/delete files and anonymous users
 will be able to read files.
 
-Additional configuration may be required by the storage adapter implementation.
+Additional java system properties and/or configuration files may be required to configure the appropriate storage adapter:
```

Comment:
move this paragraph and list about adapters up and merge with the config property; down here it looks like it's part of *For developer testing only*